.golangci.yml

```
@@ -0,0 +1,5 @@
+issues:
+  exclude-rules:
+    - path: _test\.go
+      linters:
+        - errcheck
```

should we still have it to ensure the tests aren't ignoring errors?

---

Yeah, I can add it back in.